notes/returned-mail-received.vbs

```diff
@@ -109,32 +135,43 @@ EMReadScreen case_invalid_error, 72, 24, 2 'if a person enters an invalid footer
 case_invalid_error = trim(case_invalid_error)
 If case_invalid_error = "ENTER A VALID COMMAND OR PF-KEY" Then case_invalid_error = ""			'this message is for if you press transmit on a single instance panel and does not indicate an error. Ignoring it
 IF trim(case_invalid_error) <> "" THEN script_end_procedure("*** NOTICE!!! ***" & vbCr & case_invalid_error & vbCr & "Please resolve for the script to continue.")
+
+CALL determine_program_and_case_status_from_CASE_CURR(case_active, case_pending, case_rein, family_cash_case, mfip_case, dwp_case, adult_cash_case, ga_case, msa_case, grh_case, snap_case, ma_case, msp_case, emer_case, unknown_cash_pending, unknown_hc_pending, ga_status, msa_status, mfip_status, dwp_status, grh_status, snap_status, ma_status, msp_status, msp_type, emer_status, emer_type, case_status, list_active_programs, list_pending_programs)
+
+snap_or_cash_case = False 'we need to establish if snap or cash are active, pending, for how the case is handled'
+If family_cash_case = True Then snap_or_cash_case = True
+If mfip_case = True Then snap_or_cash_case = True
```

Comment:
mfip and dwp are duplicates of family cash

Reply:
removed